### ISSUE TITLE
Add DecodeMultiple

### ISSUE THREAD

**Issue by mezlondr** [Spelt/ZXing.Delphi#165]
Hello,

please add options to decode multiple barcodes in an image to the port. Such as [MultipleBarcodeReader](https://zxing.github.io/zxing/apidocs/com/google/zxing/multi/GenericMultipleBarcodeReader.html) with decodeMultiple.

**Comment by Spelt** [comment]
Well I do except pull requests. 😀Op 20 dec 2023 om 14:14 heeft mezlondr ***@***.***> het volgende geschreven:﻿
Hello,
please add options to decode multiple barcodes in an image to the port. Such as MultipleBarcodeReader with decodeMultiple.

**Comment by RicardoFragoso99** [comment]
Hi.
Just to clarify, decoding of multiple barcodes is available or not?
If not, this is someting that is expected in near future?
Thank you

**Comment by Spelt** [comment]
HiI think you want to be able to scan multiple codes in one scan? Its not available and i’m not planning for it but i’am excepting pull requests and patches. Op 25 sep 2024 om 16:40 heeft RicardoFragoso99 ***@***.***> het volgende geschreven:﻿
Hi.
Just to clarify, decoding of multiple barcodes is available or not?
If not, this is someting that is expected in near future?
Thank you

**Comment by RicardoFragoso99** [comment]
Yes.
If a image has several barcodes, the scan should return them all in a list.
What do you mean exactly by "but i’am excepting pull requests and patches" ?

> HiI think you want to be able to scan multiple codes in one scan? Its not available and i’m not planning for it but i’am excepting pull requests and patches. Op 25 sep 2024 om 16:40 heeft RicardoFragoso99 ***@***.***> het volgende geschreven:﻿ Hi. Just to clarify, decoding of multiple barcodes is available or not? If not, this is someting that is expected in near future? Thank you —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

**Comment by Spelt** [comment]
HiThat means that everyone can add features and fixes and send the changes to me. Op 25 sep 2024 om 17:54 heeft RicardoFragoso99 ***@***.***> het volgende geschreven:﻿
Yes.
If a image has several barcodes, the scan should return them all in a list.
What do you mean exactly by "but i’am excepting pull requests and patches" ?

HiI think you want to be able to scan multiple codes in one scan? Its not available and i’m not planning for it but i’am excepting pull requests and patches. Op 25 sep 2024 om 16:40 heeft RicardoFragoso99 @.> het volgende geschreven:﻿ Hi. Just to clarify, decoding of multiple barcodes is available or not? If not, this is someting that is expected in near future? Thank you —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: @.>

**Comment by mezlondr** [comment]
> HiThat means that everyone can add features and fixes and send the changes to me. Op 25 sep 2024 om 17:54 heeft RicardoFragoso99 ***@***.***> het volgende geschreven:﻿ Yes. If a image has several barcodes, the scan should return them all in a list. What do you mean exactly by "but i’am excepting pull requests and patches" ? HiI think you want to be able to scan multiple codes in one scan? Its not available and i’m not planning for it but i’am excepting pull requests and patches. Op 25 sep 2024 om 16:40 heeft RicardoFragoso99 @.> het volgende geschreven:﻿ Hi. Just to clarify, decoding of multiple barcodes is available or not? If not, this is someting that is expected in near future? Thank you —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: @.> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

Oh I see the confusion, I think we all thought that "excepting" is a typo of "expecting", but you actually mean "accepting", am I right?

**Comment by Spelt** [comment]
You’re absolutely right. Op 26 sep 2024 om 09:20 heeft mezlondr ***@***.***> het volgende geschreven:﻿

HiThat means that everyone can add features and fixes and send the changes to me. Op 25 sep 2024 om 17:54 heeft RicardoFragoso99 @.> het volgende geschreven:﻿ Yes. If a image has several barcodes, the scan should return them all in a list. What do you mean exactly by "but i’am excepting pull requests and patches" ? HiI think you want to be able to scan multiple codes in one scan? Its not available and i’m not planning for it but i’am excepting pull requests and patches. Op 25 sep 2024 om 16:40 heeft RicardoFragoso99 @.> het volgende geschreven:﻿ Hi. Just to clarify, decoding of multiple barcodes is available or not? If not, this is someting that is expected in near future? Thank you —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: @.> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: @.>

Oh I see the confusion, I think we all though that "excepting" is a typo of "expecting", but you actually mean "accepting", am I right?
